email-manager/src/domain/msg/msg_handler.rs

```diff
@@ -161,16 +162,18 @@ pub fn mailto<
         }
     }
 
-    let mut msg = Msg::default();
+    let msg = Msg {
+        from: Some(vec![account.address().parse()?]),
+        to: if to.is_empty() { None } else { Some(to) },
+        cc: if cc.is_empty() { None } else { Some(cc) },
+        bcc: if bcc.is_empty() { None } else { Some(bcc) },
+        subject: subject.into(),
+        parts: Parts(vec![Part::TextPlain(TextPlainPart {
+            content: body.into(),
+        })]),
+        ..Default::default()
```

Comment:
Is it a common practice in Rust to use the `Default::defaut()` instead of `Msg::default()`?

Reply:
I let clippy's fix-it correct it for me and it automatically replaced `Msg` with `Default`. In all honesty, I wasn't aware of that syntax (using `..` in a struct instantiation).

I have no idea what the best practice on that is. It works with `Msg::default()` so I changed it from `Default` to `Msg`.